spec/models/organization_spec.rb

```diff
@@ -237,30 +241,36 @@
     end
 
     context "when no organization is provided" do
+      # TODO: this should probably have hard-coded numbers
       it "updates all organizations" do
-        Organization.seed_items(@organization)
-        second_organization = create(:organization)
-        organization_item_count = @organization.items.size
+        Organization.seed_items(organization)
+        organization_item_count = organization.items.size
+        base_item_count = BaseItem.count
+
+        second_organization = create(:organization, skip_items: true)
         second_organization_item_count = second_organization.items.size
+
         create(:base_item, name: "Foo", partner_key: "foo")
+
         Organization.seed_items
-        expect(@organization.items.size).to eq(organization_item_count + 1)
-        expect(second_organization.items.size).to eq(second_organization_item_count + 1)
+        organization.reload
+        second_organization.reload
+
+        expect(organization.items.size).to eq(organization_item_count + 1)
+        expect(second_organization.items.size).to eq(second_organization_item_count + base_item_count + 1)
```

Comment:
Should the numbers be hard-coded here? If so what should they be?

Reply:
Might not need to - you can use `expect { Organization.seed_items }.to change { organization.items.count }.by(1)`